tools/getkeys/index.js

```diff
@@ -64,8 +64,7 @@ function stdResponse(err, stdout, stderr) {
     try {
         // Key Vault with restricted access for the FF dev team only
         await getKeys(client, rc, "https://ff-internal-dev-secrets.vault.azure.net/");
-    } catch (e) {
-        console.log("Couldn't get secrets from FF internal keyvault. If you need access make sure you are in the relevant security group.")
-    }
+        console.log("Overriding defaults with values from the FF internal keyvault.")
+    } catch (e) {}
```

Comment:
Should we keep the original log line in the catch block but demote it to debug? Or would that still be noisy?

Reply:
I don't think it matters at all.  The presence/absence of the other log is tied definitively to this code path.